BUILD

```diff
@@ -1,241 +1,280 @@
+    "gcr.io/{PROJECT_ID}/cc-debian9:{COMMIT_SHA}": "//cc:cc_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/cc-debian10:{COMMIT_SHA}": "//cc:cc_root_amd64_debian10",
+}
+
+CC.update({
+    "gcr.io/{PROJECT_ID}/cc:" + tag_base + "-" + arch: "//cc:" + label + "_" + user + "_" + arch + "_debian9"
+    for arch in ARCHITECTURES
+    for (tag_base, label, user) in [
+        ("latest", "cc", "root"),
+        ("nonroot", "cc", "nonroot"),
+        ("debug", "debug", "root"),
+        ("debug-nonroot", "debug", "nonroot"),
+    ]
+})
+
+CC.update({
+    "gcr.io/{PROJECT_ID}/cc-" + distro + ":" + tag_base + "-" + arch: "//cc:" + label + "_" + user + "_" + arch + "_" + distro
+    for arch in ARCHITECTURES
+    for (tag_base, label, user) in [
+        ("latest", "cc", "root"),
+        ("nonroot", "cc", "nonroot"),
+        ("debug", "debug", "root"),
+        ("debug-nonroot", "debug", "nonroot"),
+    ]
+    for distro in DISTROS
+})
+
+PYTHON3 = {
+    "gcr.io/{PROJECT_ID}/python3:{COMMIT_SHA}": "//experimental/python3:python3_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/python3-debian9:{COMMIT_SHA}": "//experimental/python3:python3_root_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/python3-debian10:{COMMIT_SHA}": "//experimental/python3:python3_root_amd64_debian10",
+}
+
+PYTHON3.update({
+    "gcr.io/{PROJECT_ID}/python3:" + tag_base + "-" + arch: "//experimental/python3:" + label + "_" + user + "_" + arch + "_debian9"
+    for arch in BASE_ARCHITECTURES
+    for (tag_base, label, user) in [
+        ("latest", "python3", "root"),
+        ("nonroot", "python3", "nonroot"),
+        ("debug", "debug", "root"),
+        ("debug-nonroot", "debug", "nonroot"),
+    ]
+})
+
+PYTHON3.update({
+    "gcr.io/{PROJECT_ID}/python3-" + distro + ":" + tag_base + "-" + arch: "//experimental/python3:" + label + "_" + user + "_" + arch + "_" + distro
+    for arch in BASE_ARCHITECTURES
+    for (tag_base, label, user) in [
+        ("latest", "python3", "root"),
+        ("nonroot", "python3", "nonroot"),
+        ("debug", "debug", "root"),
+        ("debug-nonroot", "debug", "nonroot"),
+    ]
+    for distro in DISTROS
+})
+
+PYTHON27 = {
+    "gcr.io/{PROJECT_ID}/python2.7:{COMMIT_SHA}": "//experimental/python2.7:python27_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/python2.7-debian9:{COMMIT_SHA}": "//experimental/python2.7:python27_amd64_debian9",
+    "gcr.io/{PROJECT_ID}/python2.7-debian10:{COMMIT_SHA}": "//experimental/python2.7:python27_amd64_debian10",
+}
+
+PYTHON27.update({
+    "gcr.io/{PROJECT_ID}/python2.7:" + tag_base + "-" + arch: "//experimental/python2.7:" + label + "_" + arch + "_debian9"
+    for arch in BASE_ARCHITECTURES
+    for (tag_base, label) in [
+        ("latest", "python27"),
+        ("debug", "debug"),
+    ]
+})
+
+PYTHON27.update({
+    "gcr.io/{PROJECT_ID}/python2.7-" + distro + ":" + tag_base + "-" + arch: "//experimental/python2.7:" + label + "_" + arch + "_" + distro
```

Comment:
ditto for `python2.7-debian{9|10}`.

Reply:
added this to the loop, so we get the suffixes for these as well.